virttest/qemu_storage.py

```diff
@@ -828,6 +828,51 @@ def compare_to(self, target_image, source_cache_mode=None,
 
         return result
 
+    def check(self, params, root_dir, force_share=False, output=None):
+        """
+        Check an image using the appropriate tools for each virt backend.
+
+        :param params: Dictionary containing the test parameters.
+        :param root_dir: Base directory for relative filenames.
+        :param output: The format of the output(json, human).
+
+        :note: params should contain:
+               image_name -- the name of the image file, without extension
+               image_format -- the format of the image (qcow2, raw etc)
+
+        :return: The output of check result if the image exists, or None.
+        """
+        image_filename = self.image_filename
+        logging.debug("Checking image file %s", image_filename)
+        force_share &= self.cap_force_share
+
+        check_img = self.support_cmd("check") and self.support_cmd("info")
```

Comment:
The `info` command is not used at here, right?

Reply:
Seems not. But this is the original script. And from my point of view, `info` is a basic function, `check` can work only when `info` works fine.

Reply:
Updated, thanks.